src/store/modules/ProfilesModule.ts

```diff
@@ -41,6 +44,10 @@ export const ProfilesModule = {
             await dispatch('setSelectedProfile', { profileName: 'Default', prewarmCache: true });
         },
 
+        async reset({commit}) {
```

Comment:
I don't see much value in having these async actions that call the synchronous mutations when we could just call the mutations directly.